### DISCUSSION TITLE
求Undertale 64位

### DISCUSSION THREAD

**Discussion by Songbingzhi628** [Gamuxorg/bbs#423] [投递和请求游戏]
现在已有的那个我这里报错说没有32位库

**Comment by hyh19962008** [comment]
可以发一下报错信息吗

**Comment by Songbingzhi628** [comment]
我这边后来又自己修好了。主要是个别运行库64位不兼容32位要求，我这边是libcrypto.so.1.0.0，libssl.so.1.0.0，记得还有什么libpulse

在 2023-11-29 17:13:44，Demon ***@***.***> 写道：


可以发一下报错信息吗
